src/pages/docker/test/configure-xdebug.md

```diff
@@ -0,0 +1,169 @@
+
+1. Change the Xdebug port to 9001 in the **PHP** > **Debug** > **Xdebug** > **Debug Port** panel.
+
+1. Click **Apply**.
+
+## Use Xdebug
+
+The following steps describe debugging web requests and CLI commands.
+
+**To debug web requests**:
+
+1. In your PhpStorm project, click ![Start listening for connections](../../_images/xdebug/listening.png) (**Start listening**) in the top navigation bar.
+
+1. Add breakpoints in the `pub/index.php` file.
+
+1. Install the debug extension in the browser, and then click **Debug** to enable.
+
+1. In the browser, open the `https://localhost` URL.
+
+1. When PhpStorm recognizes the Xdebug connection, you can begin debugging web requests.
+
+You can debug any Adobe Commerce command or PHP script using the following steps.
+
+**To debug CLI commands**:
+
+1. In your PhpStorm project, open the **Build, Extension, Deployment** > **Docker** panel, and then click `+` to add a Docker server and update the following settings:
+
+   -  **Name**—Enter a name for the server, for example `Docker Cloud`.
+   -  **Connect to Docker daemon with**—
+      -  _Windows_—Select **TCP socket** and update **Engine Api Url** with `tcp://localhost:2375`.
+      -  _macOS X_—Select **Docker for Mac**. [_default_]
+
+1. In the **PHP** > **Cli Interpreter** panel, click **[...]**.
+
+1. Click **[+]** to add and configure a new Cli Interpreter from your Docker image. Update the following settings:
+
+   -  **Name**—Enter a name for the new interpreter, such as `Magento cloud docker cli`
+   -  **Remote**—Select `Docker`
+      -  **Server**—Select `Docker Cloud` from the previous step.
+      -  **Image name**—Select `magento/magento-cloud-docker-php:7.x-cli`
+   -  **Additional** > **Debugger extension**:
+      -  Windows—Enter `xdebug`
+      -  macOS X/Linux—Enter `xdebug.so`
+   -  Click **Refresh** to verify that the interpreter and Xdebug extension are configured properly.
+
+1. Click **Save**.
+
+1. Open the _Run/Debug Configuration_ window and add a **PHP script** with the following settings:
+
+   - **Name**—Enter `bin/magento`
+   - **Configuration** > **File**—Select the path to the `bin/magento` file in your local environment.
+
+1. Add breakpoints in the `bin/magento` file and the debug PHP script created in the previous step.
+
+## Using Xdebug Helper
+
+You can install and use the Xdebug Helper Chrome extension to debug your PhP code from the browser.
+
+**To use Xdebug Helper with Chrome**:
+
+1. Install the [Xdebug Helper extension](https://chrome.google.com/webstore/detail/xdebug-helper/eadndfjplgieldjbigjakmdgkmoaaaoc?hl=en) from the Chrome store.
+
+1. Enable the extension in Chrome as shown in the following figure.
+
+   ![Enable the Xdebug extension in Chrome](../../_images/xdebug/enable.png)
+
+1. In _Chrome_, click ![Xdebug helper icon](../../_images/xdebug/helper-icon.png) in the Chrome toolbar.
```

Comment:
It seems as though the theme wraps text when rendering images inline.

![Screenshot 2022-11-30 at 1 50 30 PM](https://user-images.githubusercontent.com/13662379/204894611-21fa707b-dbf5-4264-90dd-cf6bc95fa951.png)